plugins/wmm_pi/src/WMM_SubLibrary.c

```diff
@@ -1790,21 +1790,21 @@ WMMtype_LegendreFunction *WMM_AllocateLegendreFunctionMemory(int NumTerms)
 	if (!LegendreFunction) {
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+		return NULL;
 					}
 	LegendreFunction->Pcup = (double *) 	malloc	( (NumTerms +1) * sizeof ( double ) );
 	if (LegendreFunction->Pcup == 0)
 	{
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+        return NULL;

```

Comment:
I thought we were supposed to prefer spaces over tabs on any new commits. Previously I was chastened for allowing tabs. Now I am being chastened for not having tabs. Is there a consistent rule?